include/partition/inertial_flow.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef OSRM_PARTITION_INERTIAL_FLOW_HPP_
+#define OSRM_PARTITION_INERTIAL_FLOW_HPP_
+
+#include "partition/graph_view.hpp"
+#include <vector>
+
+namespace osrm
+{
+namespace partition
+{
+
+class InertialFlow
+{
+  public:
+    InertialFlow(const GraphView &view);
+
+    std::vector<bool> ComputePartition(const double balance, const double source_sink_rate);
```

Comment:
Should we use dynamic bitset here

http://www.boost.org/doc/libs/1_63_0/libs/dynamic_bitset/dynamic_bitset.html

(search for vector bool in the docs there)